advisor/src/main/kotlin/VulnerabilityProviderFactory.kt

```diff
@@ -31,25 +31,25 @@ interface VulnerabilityProviderFactory {
     /**
      * The name to use to refer to the advisor.
      */
-    val advisorName: String
+    val providerName: String
 
     /**
-     * Create an [Advisor] using the specified [config].
+     * Create an [VulnerabilityProvider] using the specified [config].
      */
-    fun create(config: AdvisorConfiguration): Advisor
+    fun create(config: AdvisorConfiguration): VulnerabilityProvider
 }
 
 /**
  * A generic factory class for an [Advisor].
```

Comment:
The comment refers to an [Advisor], but needs to be updated to a [VulnerabilityProvider]. This applies to multiple comments in this file, which still mention advisors.